test/LayoutTest.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import test from 'tape';
+
+import Layout from '../src/Layout';
+
+test('<Layout>', t => {
+  const wrapper = shallow(<Layout skin="blue">
+      <p>asdf</p>
+    </Layout>);
+
+  t.ok(wrapper.hasClass('sidebar-mini'), 'has (default) sidebar-mini class');
+  t.ok(wrapper.children().hasClass('wrapper'), 'renders child wrapper');
+  t.end()
+});
+
+test('<Layout> skin', t => {
+  const wrapper = shallow(<Layout skin="blue">
+      <p>asdf</p>
+    </Layout>);
+
+  t.ok(wrapper.hasClass('skin-blue'), 'is used');
```

Comment:
Since `skin` is a required property and it's already being set in the [previous test](https://github.com/jonmpqts/react-admin-lte/blob/layout/test/LayoutTest.jsx#L8) I would move this assertion up to that test and get rid of this one.